tests/test_assessment_engine.py

```diff
@@ -55,6 +56,46 @@ def test_submit_invalid_assessment(self):
         )
         assert response.status_code == 400
 
+    def test_duplicate_identifier(self):
+        swagger_spec = swagger(self.app)
+        identifier = Identifier(system='https://unique.org', value='abc123')
+        data = swagger_spec['definitions']['QuestionnaireResponse']['example']
+        data['identifier'] = identifier.as_fhir()
+
+        self.promote_user(role_name=ROLE.PATIENT.value)
+        self.login()
+        response = self.client.post(
+            '/api/patient/{}/assessment'.format(TEST_USER_ID),
+            content_type='application/json',
+            data=json.dumps(data),
```

Comment:
You can pass JSON data (and omit the `content_type='application/json'` header) with `json=data`

[See](http://flask.pocoo.org/docs/1.0/changelog/#version-1-0)
>Add json keyword argument for the test client request methods. This will dump the given object as JSON and set the appropriate content type

Reply:
thanks - neat enhancement